{"name": "Проверка возникновения ошибки при авторизации несуществующего курьера", "status": "passed", "steps": [{"name": "Авторизация созданного курьера", "status": "passed", "parameters": [{"name": "login", "value": "'Ninja'"}, {"name": "password", "value": "'1234'"}], "start": 1747235121729, "stop": 1747235121790}], "start": 1747235121729, "stop": 1747235121790, "uuid": "3ebf7df7-0fbf-439c-a502-bfbebc6413d5", "historyId": "933b925dd924b6df34a453f16ac3ea89", "testCaseId": "933b925dd924b6df34a453f16ac3ea89", "fullName": "tests.test_couriers.TestLoginCourier#test_login_not_creating_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_couriers"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-579T290"}, {"name": "thread", "value": "8216-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_couriers"}]}